dmapiclient/data.py

```diff
@@ -330,6 +330,11 @@ def export_users(self, framework_slug):
             "/users/export/{}".format(framework_slug)
         )
 
+    def email_address_has_valid_buyer_domain(self, email_address):
```

Comment:
We did, didn't we, and then I got sidetracked by a long discussion about the spacing of bullet points on a page that no-one should ever really see :/